src/fu-device.c

```diff
@@ -1411,6 +1419,44 @@ fu_device_set_logical_id (FuDevice *self, const gchar *logical_id)
 	priv->logical_id = g_strdup (logical_id);
 }
 
+/**
+ * fu_device_get_protocol:
+ * @self: A #FuDevice
+ *
+ * Gets the protocol ID on the device. This may be used in the future to verify
+ * firmware files.
+ *
+ * Returns: a string value, or %NULL if never set.
+ *
+ * Since: 1.3.5
+ **/
+const gchar *
+fu_device_get_protocol (FuDevice *self)
+{
+	FuDevicePrivate *priv = GET_PRIVATE (self);
+	g_return_val_if_fail (FU_IS_DEVICE (self), NULL);
+	return priv->protocol;
+}
+
+/**
+ * fu_device_set_protocol:
+ * @self: A #FuDevice
+ * @protocol: a defined protocol ID, e.g. `org.hughski.colorhug`
+ *
+ * Sets the protocol ID on the device. This may be used in the future to verify
+ * firmware files.
```

Comment:
I don't think future is a good thing to mention here. We'll prob forget to update this comment since the code won't be touched.